Sourcery/Sourcery.swift

```diff
@@ -19,8 +19,15 @@ import XcodeProj
 public class Sourcery {
     public static let version: String = SourceryVersion.current.value
     public static let generationMarker: String = "// Generated using Sourcery"
-    public static let generationHeader = "\(Sourcery.generationMarker) \(Sourcery.version) — https://github.com/krzysztofzablocki/Sourcery\n"
-        + "// DO NOT EDIT\n"
+    private (set) public lazy var generationHeader = {
```

Comment:
I had to change this from static since now it changes based on a flag for Sourcery. Happy to discuss alternative approaches.